src/CMakeLists.txt

```diff
@@ -1836,6 +1836,9 @@ if(DEFINED TINYSPLINE_CSHARP_FRAMEWORK_VERSION)
   endif()
   if(${TINYSPLINE_PLATFORM_ARCH} STREQUAL "arm")
     set(TINYSPLINE_NUGET_RID "${TINYSPLINE_NUGET_RID}-arm")
+    # If tinysplines should be build as a osx-universal
+  elseif ("${CMAKE_OSX_ARCHITECTURES}" STREQUAL "arm64;x86_64")
```

Comment:
`CMAKE_OSX_ARCHITECTURES` is a list. The order of the values could also be `x86_64;arm64`. You should check the size of the list and check whether it contains the values `arm64` and `x86_64`.

See https://cmake.org/cmake/help/latest/command/list.html#length and https://cmake.org/cmake/help/latest/command/list.html#search